ingestion/src/metadata/orm_profiler/profiler/sampler.py

```diff
@@ -118,10 +121,17 @@ def fetch_sqa_sample_data(self) -> TableData:
             .limit(self.sample_limit)
             .all()
         )
+        rows = [list(row) for row in sqa_sample]
+        for col_index, col in enumerate(sqa_columns):
+            if type(col.type) in [sqltypes.LargeBinary, sqltypes.BINARY]:
+                rows = [
+                    [f"{r}" if index == col_index else r for index, r in enumerate(row)]
+                    for row in rows
+                ]
```

Comment:
Instead of modifying this at runtime by iterating over the data, we should probably create a custom type and handle the conversion of the data type there -- see [here](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/orm_profiler/orm/types/bytea_to_string.py) for an example